ecl2df/gruptree.py

```diff
@@ -353,9 +430,17 @@ def gruptree_main(args) -> None:
     if args.prettyprint:
         if "DATE" in dframe:
             for date in dframe["DATE"].dropna().unique():
+                df_date = dframe[dframe.DATE == date]
                 print("Date: " + str(date.astype("M8[D]")))
-                for tree in edge_dataframe2dict(dframe[dframe["DATE"] == date]):
-                    print(tree_from_dict(tree))
+
+                for treetype in ["GRUPTREE", "BRANPROP"]:
+                    if treetype in df_date["KEYWORD"].unique():
+                        df_treetype = df_date[
+                            df_date["KEYWORD"].isin([treetype, "WELSPECS"])
+                        ]
+                        print(f"{treetype} trees:")
```

Comment:
Add test for BRANPROP tree.

Reply:
Consider whitespace between the pretty-printed trees, will it look good for all combinations of tree types?

Reply:
This is how it looks now. There is an empty line between the trees. Any suggestions?

```
Date: 2000-01-01
GRUPTREE trees:
FIELD
└── TMPL_A
    └── WELL_1

TMPL_B
└── WELL_2

BRANPROP trees:
FIELD
└── NODE_A
    └── TMPL_A
        └── WELL_1


Date: 2000-02-01
BRANPROP trees:
FIELD
└── NODE_A
    └── TMPL_A
        └── WELL_1
```

Reply:
That looks good enough for me!